crates/turbo-tasks-memory/src/scope.rs

```diff
@@ -393,11 +408,12 @@ impl TaskScope {
             "Scope dependent tasks not correctly cleaned up: {:?}",
             state.dependent_tasks
         );
-        assert!(
-            state.collectibles.is_empty(),
-            "Scope collectibles not correctly cleaned up: {:?}",
-            state.collectibles
-        );
+        // TODO read_collectibles_tasks need to be cleaned up
```

Comment:
Is there a linear issue for this?

Reply:
https://linear.app/vercel/issue/WEB-615/the-read-collectibles-in-taskscope-is-not-cleaned-up-when-the